features/validate_image.feature

```diff
@@ -471,7 +471,7 @@ Feature: evaluate enterprise contract
       ]
     }
     """
-    When ec command is run with "validate image --snapshot acceptance/happy --policy acceptance/ec-policy --rekor-url ${REKOR} --public-key ${known_PUBLIC_KEY} --strict  --show-successes"
+    When ec command is run with "validate image --snapshot acceptance/happy --policy acceptance/ec-policy --rekor-url ${REKOR} --public-key ${known_PUBLIC_KEY}   --show-successes"
```

Comment:
Nitpick: There are some extra spaces here and there, e.g. `before --show-successes` in this line and some of the others. It doesn't matter much, but you could consider could tidying them up.

Reply:
Ok think i fixed all the spacing sorry for that.